shared/transforms/pcidss/transform_benchmark_to_pcidss.py

```diff
@@ -185,6 +185,10 @@ def main():
         root_element.get("id").replace("_benchmark_", "_benchmark_PCIDSS-")
     )
 
+    for title_element in \
+            root_element.findall("./{%s}title" % (XCCDF_NAMESPACE)):
+        title_element.text += " (PCI-DSS variant)"
```

Comment:
How about ` (PCI-DSS centric)`?
Just to be consistent with suffix added for Profiles in this Benchmark.
Or change the suffix for Profiles to `_variant`.